README.md

```diff
@@ -87,39 +106,99 @@ The VTS is a metric designed to help stakers evaluate the performance and reliab
 The VTS is displayed in the Nimiq Wallet, allowing stakers to compare validators and select the one that best meets their needs.
 
 - [Read the docs](https://nimiq.com/developers/learn/validator-trustscore)
-- [See implementation](./packages/nimiq-validators-trustscore/)
+- Checkout the [pnpm package](./packages/nimiq-validator-trustscore/)
 
 ## Validators API
 
 The Validators API provides endpoints to retrieve validator information for integration with tools, dashboards, and other applications.
```

Comment:
There are missing endpoints in this table that are present in the API

Reply:
I don't want to document them since I don't want to people to use them... They might change in the future and they are just internal for us...